bigbluebutton-html5/imports/ui/components/video-provider/video-list/video-list-item/component.jsx

```diff
@@ -223,39 +223,41 @@ class VideoListItem extends Component {
           />
           {videoIsReady && this.renderFullscreenButton()}
         </div>
-        { videoIsReady &&
+        { videoIsReady
+          && (
           <div className={styles.info}>
-          {enableVideoMenu && availableActions.length >= 3
-            ? (
-              <Dropdown className={isFirefox ? styles.dropdownFireFox : styles.dropdown}>
-                <DropdownTrigger className={styles.dropdownTrigger}>
-                  <span>{name}</span>
-                </DropdownTrigger>
-                <DropdownContent placement="top left" className={styles.dropdownContent}>
-                  <DropdownList className={styles.dropdownList}>
-                    {availableActions}
-                  </DropdownList>
-                </DropdownContent>
-              </Dropdown>
-            )
-            : (
-              <div className={isFirefox ? styles.dropdownFireFox
-                : styles.dropdown}
-              >
-                <span className={cx({
-                  [styles.userName]: true,
-                  [styles.noMenu]: numOfStreams < 3,
-                })}
+            {enableVideoMenu && availableActions.length >= 3
+              ? (
+                <Dropdown ignoreMobile className={isFirefox ? styles.dropdownFireFox : styles.dropdown}>
```

Comment:
Adding `tethered` as a prop on the `Dropdown` and `height: 90%` the class `dropdownContent` will allow you to make the menu take up the entire screen.
  
![video-item-dropdown](https://user-images.githubusercontent.com/22058534/113210623-6ed6cd80-9242-11eb-8811-a7f967151c6f.gif)

Reply:
Thanks @KDSBrowne! This looks a lot better than changing 7 files :) 
I'll close this PR a send a new one with your suggestion